src/python/deepops/cli.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+
+"""Console script for deepops."""
+import sys
+import click
+import os
+
+from .repo import clone_repo, local_repo_path
+from .deps import run_deepops_setup
+from .ansible import (
+    make_ansible_inventory_file,
+    make_host_groups_for_local,
+    make_ansible_vars_file,
+    run_ansible_playbook,
+    AnsibleFailedError,
+)
+
+
+@click.group()
+def main(args=None):
+    """GPU infrastructure and automation tools"""
+    return 0
+
+
+#######################################################################
+# Repository management commands
+#######################################################################
+
+
+@main.group()
+def repo(args=None):
+    """Commands for managing the local copy of the DeepOps repo"""
+    return 0
+
+
+@repo.command()
+@click.option("--path", default=None)
+@click.option(
+    "--remote", default="https://github.com/NVIDIA/deepops", show_default=True
+)
+@click.option("--tag", default="20.06", show_default=True)
+@click.option("--force", is_flag=True)
+def clone(path, remote, tag, force):
+    """Clone DeepOps repository locally"""
+    clone_repo(repo_path=path, remote=remote, force=force, tag=tag)
+
+
+@repo.command()
+def show_path():
+    """Show configured repository path"""
+    click.echo(local_repo_path())
+
+
+#######################################################################
+# Install commands
+#######################################################################
+
+
+@main.group()
+def install(args=None):
+    """Commands for installing DeepOps components"""
+    return 0
+
+
+@install.command(name="deepops-dependencies")
+def deepops_deps():
+    click.echo("Installing DeepOps repo dependencies")
+    run_deepops_setup()
+
+
+@install.command(name="nvidia-driver")
+@click.option("--debug", is_flag=True)
+@click.option("--dry-run", is_flag=True)
+def nvidia_driver(debug, dry_run):
+    """Install NVIDIA driver"""
+    click.echo("Run Ansible to install NVIDIA driver")
+    inv_file = make_ansible_inventory_file()
+    if debug:
+        click.echo("inventory file: {}".format(inv_file))
+    if dry_run:
+        click.echo(
+            "Would have run ansible-playbook with {}/playbooks/nvidia-driver.yml".format(
+                local_repo_path()
+            )
+        )
+        return
+    run_deepops_setup()
```

Comment:
It'd be nice to find a way to not run this every time, maybe write/check a file in `~/.local/share/deepops/`?